src/Group_User.php

```diff
@@ -493,7 +492,28 @@ public static function showForGroup(Group $group)
 
        // Retrieve member list
        // TODO: migrate to use CommonDBRelation::getListForItem()
-        $entityrestrict = self::getDataForGroup($group, $used, $ids, $crit, $tree, false);
+        $entityrestrict = self::getDataForGroup($group, $used, $ids, $crit, 0, false);
```

Comment:
The `tree` parameter (to show child groups) is not anymore passed here. It will probably break the `Child groups: Yes` case.

Reply:
Not sure why I changed this, probably something I forgot to revert.
Done `6a587d4ef6`.